<launch>

	<!-- Load controller manager -->
	<rosparam command="load" file="$(find iai_robots_control)/config/default_controllers.yaml" />
	<node name="ros_control_controller_manager" pkg="controller_manager" type="controller_manager" respawn="false" output="screen" args="spawn 
		joint_state_controller
		ur5_shoulder_pan_joint_position_controller
		ur5_shoulder_lift_joint_position_controller
		ur5_elbow_joint_position_controller
		ur5_wrist_1_joint_position_controller
		ur5_wrist_2_joint_position_controller
		ur5_wrist_3_joint_position_controller
		wheel_back_left_joint_velocity_controller
		wheel_back_right_joint_velocity_controller
		wheel_front_left_joint_velocity_controller
		wheel_front_right_joint_velocity_controller
		gripper_joint_effort_controller
		"/>

	<node name="command_publisher" pkg="iai_robots_control" type="command_publisher.py" />
	<node name="joint_state_publisher_gui" pkg="joint_state_publisher_gui" type="joint_state_publisher_gui">
		<remap from="joint_states" to="desired_joint_states" />
	</node>

</launch>